src/openforms/utils/templatetags/utils.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag(name="get_value")
+def get_value(dict, value_key):
+    return dict.get(value_key, "")
```

Comment:
```suggestion
def get_value(value: dict, value_key):
    return value.get(value_key, "")
```
shadowing the builtin `dict` looks a bit odd. I'd also like to see some tests for non-dict values, people are definitely going to use this the wrong way. Try this on strings, lists...

I think the behaviour on non-compatible types should be "return empty string" (which is different from the normal behaviour of "let template tags crash")